gcloud-report/tests.cloudbuild.yaml

```diff
@@ -0,0 +1,92 @@
+steps:
```

Comment:
retries would be nice to add

Reply:
Good reminder. We haven't been doing that in this repo, do you think it would make sense to build the retry logic into the shared script or write it directly into each sample?

Reply:
We could add https://gist.github.com/averikitsch/961ff4e99395adcf1bcb1849fc1a4020 as a shared utility like the get service url and id token.

Reply:
Filed this for a follow-up: https://github.com/GoogleCloudPlatform/cloud-run-samples/issues/68